modules/setup/config/qtpylib_methods.py

```diff
@@ -127,17 +136,17 @@ def heikinashi(bars):
                               'low': bars['ha_low'],
                               'close': bars['ha_close']})
 
+
 # ---------------------------------------------
 
 
 def tdi(series, rsi_lookback=13, rsi_smooth_len=2,
         rsi_signal_len=7, bb_lookback=34, bb_std=1.6185):
-
     rsi_data = rsi(series, rsi_lookback)
     rsi_smooth = sma(rsi_data, rsi_smooth_len)
     rsi_signal = sma(rsi_data, rsi_signal_len)
 
-    bb_series = bollinger_bands(rsi_data, bb_lookback, bb_std)
+    bb_series = bollinger_bands(rsi_data, bb_lookback, int(bb_std))
```

Comment:
```suggestion
    bb_series = bollinger_bands(rsi_data, bb_lookback, float(bb_std))
```

bb_std is default valued at 1.6185 so it should be converted to a float, not an int!